source/mainnet/docs/protocol/manage-accounts.rst

```diff
@@ -146,16 +146,40 @@ also be set manually.
 Account aliases
 ---------------
 
-In protocol versions 1 and 2 accounts and account addresses have a one-to-one relationship. In protocol version 3 each account has 16777216 addresses, namely a so-called canonical account address together with
-matching :term:`account aliases<alias>`. The canonical account address is derived when an account is created on chain. The other 16 million addresses with matching initial 29 bytes are referred to as account aliases for
-the same account. Thus, accounts can be referred to by any address whose initial 29 bytes match.
+In protocol versions 1 and 2 accounts and account addresses have a one-to-one relationship. In protocol version 3 each account has ``16777216`` addresses, namely a so-called canonical account address together with
+matching :term:`account aliases<alias>`. The canonical account address is derived when an account is created on chain. The other 16 million addresses with matching initial ``29 bytes`` are referred to as account aliases for
+the same account. Thus, accounts can be referred to by any address whose initial ``29 bytes`` match.
 
 This allows each account to have aliases for different uses and creates a kind of sub-account structure. An account owner can give out different aliases for different uses to keep track of transfers and assign them meaning.
 
 Each account still has one total account balance. Hence, transfers to and from aliases of an account add to and subtract from that total account balance, respectively. Transfers between different aliases of the same account do not change the balance of the account, apart from cost. Rewards are always received on the account's canonical address.
 
 To show aliases, :ref:`run a transaction in Concordium Client<account-aliases>`.
 
+How account aliases work
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Concordium account addresses are ``32 bytes`` long with a specific structure:
+
+- Bytes ``0-28`` (``29`` bytes): **Account identifier** - identical for all aliases of the same account
```

Comment:
Not sure if we properly defined/named these parts consistently anywhere. In the node code base, the first ``29`` bytes are the `canonicalAccountAddress`. 

My earlier comments are properly a bit wrong that `alias 0 is the canonical account address as I was not aware of the naming in the node when adding these comments`:
https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/examples/cis2-multi/src/lib.rs#L831